assets/src/apps/delivery/components/TextParser.ts

```diff
@@ -0,0 +1,115 @@
+import { defaultGlobalEnv, evalScript, getAssignScript, getValue } from 'adaptivity/scripting';
```

Comment:
I wanted to move this new file in Utils/ directory but for some reason, it was giving webpack compile error.